gravatar/gravatar-tests.ts

```diff
@@ -0,0 +1,14 @@
+/// <reference path="gravatar.d.ts" />
+
+var gravatar = require('gravatar');
```

Comment:
Done. Why, though? Plenty of modules in the projects use this syntax. If you choose to migrate to the new syntax at some point, you’ll do it en-masse, no?